tests/test_custom_checkers/test_invalid_name_checker.py

```diff
@@ -639,7 +639,7 @@ def snake_case_format():
             self.checker.visit_functiondef(functiondef_node)
 
 
-def test_module_name_no_snippet() -> None:
+def test_module_name_no_snippet(prevent_webbrowser_and_httpserver) -> None:
```

Comment:
This test is a bit different than the others. It should be possible to modify the test to pass in the default test pylintrc file, which I believe uses a different reporter than the HTML reporter.